samples/test/metrics_visualization_v2_test.py

```diff
@@ -195,18 +179,13 @@ def verify(t: unittest.TestCase, run: kfp_server_api.ApiRun,
                     'name': 'html_artifact',
                     'type': 'system.HTML'
                 }],
-                'parameters': {}
             },
             'state': Execution.State.COMPLETE,
             'type': 'system.ContainerExecution'
         }, html_visualization.get_dict())
 
     t.assertEqual(
         {
-            'inputs': {
```

Comment:
Will it be more clear that we don't delete these so that people know this pipeline won't have any input artifacts and parameters?

Reply:
it's a trade off between length and "clarity of absence". I am leaning towards length here, but maybe we can only keep the inputs empty object.
Let me have a try in a separate pr

Reply:
I updated in this PR, what do you think?

Reply:
lgtm